src/enums/sessionStorageKeys.ts

```diff
@@ -0,0 +1,12 @@
+export enum SessionStorageKeys {
```

Comment:
We shouldn't need this. The shared components should make use of this code in their repo, not ours.

Reply:
I debated about adding this one, as resolving the magic-string antipattern for property-keys can be a bit of a russian-nesting-doll issue - but its the keys that need to be specifically set here, and this resolves that antipattern with this usage - I figured if we were to implement a means of addressing it - it might as well be the same format as used with the common-components.

I could remove it without having that significant of an impact on our codebase though, as we're only interacting with these specific keys in limited locations - however, generally if its repeated more than once, and either exposes or consumes an external dependancy - thats generally the use-case for when we'd want to implement this type of fix, no ?

Reply:
Reach into sbc-common-components package to ref this enum where you need it.

Reply:
Resolved in latest commit